app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
This is a good example of where an extra `else` statement isn't necessary. The `return 0` statement can be moved out of the `else` block and placed at the end of the function. This will make the code easier to read and understand. (Checklist: Make Code Easier - 1)